controllers/util/k8sutil.go

```diff
@@ -75,8 +75,13 @@ const (
 	StorageConsumerMirroringInfoAnnotation = "ocs.openshift.io/consumer-mirroring-info"
 	ForceDeletionAnnotationKey             = "ocs.openshift.io/force-deletion"
 	RookForceDeletionAnnotationKey         = "rook.io/force-deletion"
+	BackwardCompatabilityInfoAnnotationKey = "ocs.openshift.io/backward-compatability-info"
 )
 
+type BackwardCompatabilityInfo struct {
+	InternalConsumerUID string `json:"uid"`
```

Comment:
1. I don't think the name is descriptive enough. we need something that state that this is a pre 4.19 internal consumer UID

2. the json name should reflect that as well

```suggestion
	Pre4_19InternalConsumer string `json:"pre4_19InternalConsumer"`
```